share/completions/qdbus.fish

```
@@ -0,0 +1,36 @@
+function __fish_qdbus_complete
+    set -l argv (commandline --cut-at-cursor --tokenize)
+    argparse --name qdbus system 'bus=' literal help -- $argv
```

This is a completion, so you most likely want to silence errors.

```fish
argparse ... 2>/dev/null
or return
```